### ISSUE TITLE
While using cx_Freeze with python click framework .exe doesn't work on another windows machine 

### ISSUE THREAD

**Issue by harshkumar-dev** [marcelotduarte/cx_Freeze#1397]
**Describe the bug**
I want to create an exe  for a click based python project.  EXE works fine on the windows machine where I create them. But when I copy the msi or exe to other windows machine, it doesn't work. 

**To Reproduce**
**on Window server 2019 machine 1:**

```
git clone https://github.com/drorata/mwe-subcommands-click
cd mwe-subcommands-click
```

This is the directory structure:

```

├── README.md
├── __init__.py
├── __pycache__
│   └── hello_world.cpython-38.pyc
├── environment.yml
├── group1
│   ├── __init__.py
│   ├── __pycache__
│   │   ├── __init__.cpython-38.pyc
│   │   └── commands.cpython-38.pyc
│   └── commands.py
├── group2
│   ├── __init__.py
│   ├── __pycache__
│   │   ├── __init__.cpython-38.pyc
│   │   └── commands.cpython-38.pyc
│   └── commands.py
├── hello_world.py
├── my_cli.py
├── requirements.txt
└── setup.py
```

create requirements.txt and setup.py with following. 

cat setup.py

```
from cx_Freeze import Executable, setup

executables = [Executable("my_cli.py")]

setup(
    name="my_cli",
    version="0.1",
    description="demo cx_Freeze working with click",
    executables=executables,
)
```

cat requirements.txt

```
click==8.0.4
cx-Freeze==6.10
importlib-metadata==4.11.2
zipp==3.7.0
```

```
python -m venv venv
.\venv\Scripts\activate
pip install -r requirements.txt
python setup.py build
```

copy mwe-subcommands-click\build\exe.win-amd64-3.8 to another windows machine 2

**on widnows server 2019 machine 2**
```
PS C:\Users\opc\Desktop\exe.win-amd64-3.8\exe.win-amd64-3.8> dir


    Directory: C:\Users\opc\Desktop\exe.win-amd64-3.8\exe.win-amd64-3.8


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----         3/3/2022  10:28 PM                lib
-a----         3/3/2022  10:19 PM          13824 my_cli.exe
-a----         5/3/2021  11:54 AM          59568 python3.dll
-a----         5/3/2021  11:54 AM        4211376 python38.dll
PS C:\Users\opc\Desktop\exe.win-amd64-3.8\exe.win-amd64-3.8> .\my_cli.exe --help
PS C:\Users\opc\Desktop\exe.win-amd64-3.8\exe.win-amd64-3.8> .\my_cli.exe foo
PS C:\Users\opc\Desktop\exe.win-amd64-3.8\exe.win-amd64-3.8> .\my_cli.exe bar
```

As you can see exe prints nothing. For expected output, please see below:

**Expected behavior**
This correct works on windows server 2019 Machine 1. 

```
PS C:\Users\opc\mwe-subcommands-click\build\exe.win-amd64-3.8> .\my_cli.exe --help
Usage: my_cli.exe [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  bar
  foo
  hello-world
PS C:\Users\opc\mwe-subcommands-click\build\exe.win-amd64-3.8> .\my_cli.exe foo
running `foo` from group2/commands
PS C:\Users\opc\mwe-subcommands-click\build\exe.win-amd64-3.8> .\my_cli.exe bar
running `bar` from group1/commands.py
```

**Screenshots**

**Desktop (please complete the following information):**
 - Platform information windows server 2019:
 - OS architecture (e.g. intel):
 - cx_Freeze version [e.g. 6.10]:
 - Python version [e.g. 3.8.10]:

**Additional context**

**Comment by marcelotduarte** [comment]
Please check the [FAQ](https://cx-freeze.readthedocs.io/en/latest/faq.html#microsoft-visual-c-redistributable-package).

**Comment by harshkumar-dev** [comment]
Thank you for your response. In my original project, I did have   'include_msvcr': True. I tried following in mwe project above and it worked fine. 

from cx_Freeze import Executable, setup

executables = [Executable("my_cli.py")]
build_exe_options = {
    'include_msvcr': True
}
setup(
    name="my_cli",
    version="0.1",
    description="demo cx_Freeze working with click",
    executables=executables,
    options={'build_exe': build_exe_options},
)


I will look more in my project and create a new issue if required. Closing this for now.
